nextcord/types/interactions.py

```diff
@@ -248,19 +248,23 @@ class InteractionResponse(_InteractionResponseOptional):
     type: InteractionResponseType
 
 
-class MessageInteraction(TypedDict):
-    id: Snowflake
-    type: InteractionType
-    name: str
-    user: User
-
-
 class _EditApplicationCommandOptional(TypedDict, total=False):
     description: str
     options: Optional[List[ApplicationCommandOption]]
     type: ApplicationCommandType
 
 
+class _MessageInteractionOptional(TypedDict, total=False):
+    member: Member
+
+
+class MessageInteraction(_MessageInteractionOptional):
+    id: Snowflake
+    type: InteractionType
+    name: str
+    user: User
```

Comment:
These classes should be moved above `_EditApplicationCommandOptional`, so that it's together with `EditApplicationCommand` and also to reduce the PR diff.

Reply:
interesting, I thought they already were

Reply:
looks like I did an oops on reading the original doc